packages/platform-core/src/object_accumulator.ts

```diff
@@ -30,7 +30,11 @@ export class ObjectAccumulator<T> {
       if (Array.isArray(existingValue)) {
         return existingValue.concat(incomingValue);
       }
-      if (existingValue && typeof existingValue !== 'object') {
+      if (
```

Comment:
there should be a corresponding unit test for this

Reply:
Ah actually the change here can be removed since we are not using `ObjectAccumulator` for stack output anymore.